README.md

```diff
@@ -1,11 +1,5 @@
 # Chat Demo Application Overview
 
-## This is Kotlin version of the Demo app
-
-For the Java version you can look [here](https://github.com/twilio/twilio-chat-demo-android/tree/java).
-
-This demo app SDK version: ![](https://img.shields.io/badge/SDK%20version-3.1.0-blue.svg)
```

Comment:
Please don't remove, just update version.

Also keep link to Java version - peopel still ask about it. Need to update java branch version too, if possible.

Reply:
JFYI, this does not seem to be a problem - users report using version 3.0.1 while this one links to 3.1.0, so there's no correlation.

Reply:
fixed